src/EnergyPlus/UnitarySystem.cc

```diff
@@ -9036,7 +9032,7 @@ namespace UnitarySystems {
                                                       CompressorONFlag);
                         PartLoadRatio = HeatPLR;
                     } else if (state.dataGlobal->DoCoilDirectSolutions && state.dataUnitarySystems->HeatingLoad &&
-                               this->m_HeatingCoilType_Num == DataHVACGlobals::CoilDX_MultiSpeedHeating) {
+                               this->m_CoolingCoilType_Num == DataHVACGlobals::CoilDX_MultiSpeedHeating) {
```

Comment:
@rraustad Hi, Richard, just want to point out that in this commit I reverted two lines of codes to eliminate changes unrelated to my added feature. But these two lines look very suspicious, could you double check if these are bugs?

Reply:
Yes, this looks wrong.

Reply:
Actually, this one was here before this branch.